src/Interpreters/executeDDLQueryOnCluster.cpp

```diff
@@ -14,11 +14,10 @@
 #include <DataTypes/DataTypesNumber.h>
 #include <DataTypes/DataTypeString.h>
 #include <DataTypes/DataTypeNullable.h>
-#include <DataStreams/NullBlockOutputStream.h>
-#include <DataStreams/NullAndDoCopyBlockInputStream.h>
-#include <DataStreams/copyData.h>
+#include <Processors/NullSink.h>
```

Comment:
@KochetovNicolai `EmptySink` should be located in separate file.
Also `Sinks` should be located in dedicated `Sinks` directory.